tests/integration/targets/aws_acm/aliases

```diff
@@ -1,5 +1,5 @@
 # https://github.com/ansible/ansible/issues/67788
-unstable
+# unstable
```

Comment:
@markuman I'm unclear what to do about this label.
It looks like when `unstable` is set, the tests do not execute. But isn't this a gating criteria for the PR?
On the other hand, if the module is known to have issue such as the referenced issue, how can we test?

I've also discovered a new intermittent issue: https://github.com/ansible-collections/amazon.aws/issues/622

Reply:
ping @markuman.

Reply:
What should I do next? Should I disable these integration tests because there is a bug in amazon.aws?

Here is a fix for amazon.aws: https://github.com/ansible-collections/amazon.aws/pull/646, but now I'm concerned I'm going to have to wait until there is a new release of amazon.aws that includes a fix?

Reply:
AFAIK the next amazon.aws release is scheduled one week before community.aws will release its next version.   
imo, I would try to wait this two weeks if that solves the Issue with the CI.

Reply:
Ouch. This is not encouraging.  Given you are planning to release community.aws on Feb 14th and you are suggesting to wait 2 weeks, does that mean this PR wouldn't make it for 3.1?

Reply:
No. afaik amazon.aws is planed to release one week before community.aws.  
Means Feb 7th.  
But since I'm not a maintainer of amazon.aws, I cannot guarantee that. That's just something I've noticed on IRC/Matrix.  

Missing the release train happens more often than one would like. There will be 3.2.0 soon I think.